.github/workflows/releaseutils.yml

```diff
@@ -0,0 +1,25 @@
+name: Add sbom
+
+on:
+  workflow_dispatch:
+  workflow_run:
+        workflows: [test-latest-release]
+        types: [completed]
+
+jobs:
+    add-sbom:
+        name: Add SBOM
+        runs-on: ubuntu-latest
+        if: ${{github.event.workflow_run.conclusion=='success'}}
+        steps:
+            - name: checkout code
+              uses: actions/checkout@v4
+
+            - uses: advanced-security/sbom-generator-action@v0.0.1
+              id: sbom
+              env: 
+                    GITHUB_TOKEN: ${{ github.token }}
+            - uses: actions/upload-artifact@v3.1.0
+              with: 
+                path: ${{steps.sbom.outputs.fileName }}
+                name: "SBOM"
```

Comment:
We will see less red if there is a terminal linebreak here.

Reply:
sure, missed that

Reply:
I still see the red "no eol here" icon.
<img width="59" alt="image" src="https://github.com/user-attachments/assets/b7cdbad4-8f4b-464b-b7e4-b179a70f5595">

Reply:
I think i didn't leave a line at the end of the file , is that it?

Reply:
I made another commit does it look good now?

Reply:
yes